package exception

import "errors"

var (
	ErrAIResponse = errors.New("AI response error")
)
